gpii/node_modules/solutionsRegistry/src/js/generate-schemas.js

```diff
@@ -0,0 +1,119 @@
+/*
+
+    A script to generate a combined settings schema that includes detailed schemas for all supported settings.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+var mkdirp = require("mkdirp");
+var fs = require("fs");
+var path = require("path");
+
+// Add the ability to require JSON5 files.
+require("json5/lib/register");
+
+// Require universal so that we can resolve paths to it and any sub-modules.
+require("../../../../../");
```

Comment:
See above for deep requires

Reply:
Done, found another one in addition to the two you flagged.